workflow/rocoto/gfs_cycled_xml.py

```diff
@@ -24,6 +24,17 @@ def get_cycledefs(self):
         sdate_str = sdate.strftime("%Y%m%d%H%M")
         strings.append(f'\t<cycledef group="gdas">{sdate_str} {edate_str} {interval_str}</cycledef>')
 
+        if self._app_config.do_jedisnowda:
+            sdate_snocvr = self._base['SDATE']
+            edate_snocvr = self._base['EDATE']
+            interval_snocvr = to_timedelta('24H')
+            sdate_snocvr = sdate_snocvr + interval_snocvr
+            sdate_snocvr_str = sdate_snocvr.replace(hour=0, minute=0, second=0).strftime("%Y%m%d%H%M")
+            edate_snocvr_str = edate_snocvr.strftime("%Y%m%d%H%M")
+            interval_str = timedelta_to_HMS(interval_snocvr)
+            if sdate_snocvr <= edate_snocvr:
```

Comment:
OK, since this will only ever run on the 00Z cycle, that makes this easier.  Try
```suggestion
            if sdate_snocvr.date() <= edate_snocvr.date():
```

Reply:
Thanks @DavidHuber-NOAA for your solutions. It works perfectly. Please see below:

> rocotostat -d C96snowda_bufrtest4.db -w C96snowda_bufrtest4.xml
```
       CYCLE                    TASK                       JOBID               STATE         EXIT STATUS     TRIES      DURATION
================================================================================================================================
202112201200           gdas_stage_ic                     1525675              QUEUED                   -         0           0.0
202112201200          gdas_fcst_seg0                           -                   -                   -         -             -
202112201200    gdas_atmos_prod_f000                           -                   -                   -         -             -
202112201200    gdas_atmos_prod_f003                           -                   -                   -         -             -
202112201200    gdas_atmos_prod_f006                           -                   -                   -         -             -
202112201200    gdas_atmos_prod_f009                           -                   -                   -         -             -
================================================================================================================================
202112201800               gdas_prep                           -                   -                   -         -             -
202112201800               gdas_anal                           -                   -                   -         -             -
202112201800             gdas_sfcanl                           -                   -                   -         -             -
202112201800           gdas_analcalc                           -                   -                   -         -             -
202112201800            gdas_snowanl                           -                   -                   -         -             -
202112201800           gdas_analdiag                           -                   -                   -         -             -
202112201800          gdas_atmanlupp                           -                   -                   -         -             -
202112201800         gdas_atmanlprod                           -                   -                   -         -             -
202112201800          gdas_fcst_seg0                           -                   -                   -         -             -
202112201800    gdas_atmos_prod_f000                           -                   -                   -         -             -
202112201800    gdas_atmos_prod_f003                           -                   -                   -         -             -
202112201800    gdas_atmos_prod_f006                           -                   -                   -         -             -
202112201800    gdas_atmos_prod_f009                           -                   -                   -         -             -
202112201800            gdas_fit2obs                           -                   -                   -         -             -
202112201800            gdas_verfozn                           -                   -                   -         -             -
202112201800            gdas_verfrad                           -                   -                   -         -             -
202112201800            gdas_vminmon                           -                   -                   -         -             -
202112201800               gdas_arch                           -                   -                   -         -             -
202112201800            gdas_cleanup                           -                   -                   -         -             -
================================================================================================================================
202112210000               gdas_prep                           -                   -                   -         -             -
202112210000               gdas_anal                           -                   -                   -         -             -
202112210000             gdas_sfcanl                           -                   -                   -         -             -
202112210000           gdas_analcalc                           -                   -                   -         -             -
202112210000      gdas_prepsnowcover                           -                   -                   -         -             -
202112210000            gdas_snowanl                           -                   -                   -         -             -
202112210000           gdas_analdiag                           -                   -                   -         -             -
202112210000          gdas_atmanlupp                           -                   -                   -         -             -
202112210000         gdas_atmanlprod                           -                   -                   -         -             -
202112210000          gdas_fcst_seg0                           -                   -                   -         -             -
202112210000    gdas_atmos_prod_f000                           -                   -                   -         -             -
202112210000    gdas_atmos_prod_f003                           -                   -                   -         -             -
202112210000    gdas_atmos_prod_f006                           -                   -                   -         -             -
202112210000    gdas_atmos_prod_f009                           -                   -                   -         -             -
202112210000            gdas_fit2obs                           -                   -                   -         -             -
202112210000            gdas_verfozn                           -                   -                   -         -             -
202112210000            gdas_verfrad                           -                   -                   -         -             -
202112210000            gdas_vminmon                           -                   -                   -         -             -
202112210000               gdas_arch                           -                   -                   -         -             -
202112210000            gdas_cleanup                           -                   -                   -         -             -
================================================================================================================================
202112210600               gdas_prep                           -                   -                   -         -             -
202112210600               gdas_anal                           -                   -                   -         -             -
202112210600             gdas_sfcanl                           -                   -                   -         -             -
202112210600           gdas_analcalc                           -                   -                   -         -             -
202112210600            gdas_snowanl                           -                   -                   -         -             -
202112210600           gdas_analdiag                           -                   -                   -         -             -
202112210600          gdas_atmanlupp                           -                   -                   -         -             -
202112210600         gdas_atmanlprod                           -                   -                   -         -             -
202112210600          gdas_fcst_seg0                           -                   -                   -         -             -
202112210600    gdas_atmos_prod_f000                           -                   -                   -         -             -
202112210600    gdas_atmos_prod_f003                           -                   -                   -         -             -
202112210600    gdas_atmos_prod_f006                           -                   -                   -         -             -
202112210600    gdas_atmos_prod_f009                           -                   -                   -         -             -
202112210600            gdas_fit2obs                           -                   -                   -         -             -
202112210600            gdas_verfozn                           -                   -                   -         -             -
202112210600            gdas_verfrad                           -                   -                   -         -             -
202112210600            gdas_vminmon                           -                   -                   -         -             -
202112210600               gdas_arch                           -                   -                   -         -             -
202112210600            gdas_cleanup                           -                   -                   -         -             -
```
I have updated and committed the changes. Thank you again.